kaybee/internal/tasks/tasks_test.go

```diff
@@ -10,13 +10,8 @@ import (
 //
 // FIXTURES
 //
-func getConfig() conf.Configuration {
-	p, err := conf.NewParser("../../testdata/conf/kaybeeconf.yaml")
-	// p.SetConfigFile()
-	if err != nil {
-		return conf.Configuration{}
-	}
-	c, _ := p.Parse()
+func getConfig() conf.Config {
```

Comment:
As above: difficult to grasp the distinction Config vs Configuration

Reply:
> Yes, I agree with the complexity of that the configuration versioning introduces. I think it would be much better if the configuration evolves with the code rather so it is "unstable" until the next release.
> 
> This would allow for more flexibility I think & allow the tasks to be directly linked to the configuration file (which simplifies the development a lot)

Not sure I understand what you mean: you think we should drop the double-layered ConfigV1 + Config? I think that is still good to have, because the schema of the config could change (while the information content stays the same); that also helps by having structs dedicated to yaml unmarshaling (with the ugly exported attributes) confined in the conf package.